pyfr/integrators/dual/pseudo/__init__.py

```diff
@@ -1,12 +1,45 @@
 # -*- coding: utf-8 -*-
 
+from pkg_resources import resource_listdir
 import re
 
 from pyfr.integrators.dual.pseudo.pseudocontrollers import BaseDualPseudoController
-from pyfr.integrators.dual.pseudo.pseudosteppers import BaseDualPseudoStepper
+from pyfr.integrators.dual.pseudo.pseudosteppers import BaseDualPseudoStepper, DualDenseRKPseudoStepper
 from pyfr.integrators.dual.pseudo.multip import DualMultiPIntegrator
 from pyfr.util import subclass_where
 
+# Create subclasses for all tabulated dense schemes
```

Comment:
Move into a function and call this function; avoids polluting the global namespace.  Store the schemes as a member of the function (yes, that works).